dashboard/app/controllers/script_levels_controller.rb

```diff
@@ -400,6 +400,11 @@ def present_level
     )
     readonly_view_options if @level.channel_backed? && params[:version]
 
+    # Add video generation URL for only the last level of Dance
+    # If we eventually want to add video generation for other levels or level
+    # types, this is the condition that should be extended.
+    replay_video_view_options if @level.channel_backed? && @game.app == Game::DANCE
```

Comment:
@islemaster is this the best way to identify that we're on a project-backed dance level?

Reply:
I used
```rb
@level.is_a?(Dancelab) && @level.try(:free_play?)
```
inside `channel_backed?` to decide whether it'd be a channel-backed level.  That free-play part is a little bit of a hack though; using `channel_backed?` is probably a better call.  I'm indifferent re: `@level.is_a?` or `@game.app`.